Documentation/network/lb-ipam.rst

```diff
@@ -0,0 +1,382 @@
+.. only:: not (epub or latex or html)
+
+    WARNING: You are looking at unreleased Cilium documentation.
+    Please use the official rendered version released here:
+    https://docs.cilium.io
+
+.. _lb_ipam:
+
+********************************************
+LoadBalancer IP Address Management (LB IPAM)
+********************************************
+
+LB IPAM is a feature that allows Cilium to assign IP addresses to Services of
```

Comment:
Can you somehow word that these go in the ExternalIPs field?

Reply:
But they don't go in the ExternalIPs field. They go in the `.status.ingresses` field